docs/user_guide/representing/first_step.md

```diff
@@ -0,0 +1,151 @@
+banner = BannerDoc(
+    image_url="https://example.com/image.png",
+    title="Hello World",
+    description="This is a banner",
+)
+
+assert banner.image_url == "https://example.com/image.png"
+assert banner.title == "Hello World"
+assert banner.description == "This is a banner"
+```
+
+
+
+
+## `BaseDoc` is a Pydantic `BaseModel`
+
+The class [BaseDoc][docarray.base_doc.doc.BaseDoc] inherits from pydantic [BaseModel](https://docs.pydantic.dev/usage/models) from Pydantic. So you can use
+all the features of `BaseModel` in your `Doc` class. 
+
+This namely means that `BaseDoc`:
+
+* Will perform data validation: `BaseDoc` will check that the data you pass to it is valid. If not, it will raise an
+  error. Data being "valid"  is actually defined by the type used in the docstring itself, but we will come back to this concept later (TODO add typing section)
+
+* Can be configured using a nested `Config` class, see pydantic [documentation](https://docs.pydantic.dev/usage/model_config/) for more details on what kind of config Pydantic offer.
+
+* Can be used as a drop-in replacement for `BaseModel` in your code and is compatible with tools using Pydantic like [FastAPI]('https://fastapi.tiangolo.com/').
+
+
+###  What is the difference with Pydantic `BaseModel`? (INCOMPLETE)
+
+LINK TO THE VERSUS (not ready)
+
+[BaseDoc][docarray.base_doc.doc.BaseDoc] is not only a [BaseModel](https://docs.pydantic.dev/usage/models), 
+
+* it allows to be used with DocArray [Typed](docarray.typing) that are oriented toward MultiModal (image, audio, ...) data and for 
+Machine Learning use case TODO link the type section. 
+
+Another difference is that [BaseDoc][docarray.base_doc.doc.BaseDoc] has a generated by default `id` field that is used to uniquely identify a document.
+
+
+
+## `BaseDoc` allows to represent MultiModal and nested Data.
+
+Let's say you want to represent a Youtube video in your application. Maybe to build a search system for Youtube video.
+A Youtube video is not only composed of a video, but it also has a title, a description, a thumbnail (and more but let's keep it simple).
+
+All of these elements are from different `modalities` LINK TO MODALITIES SECTION (not ready), title and description are text, the thumbnail is an image, and the video in itself is, well, a video.
+
+DocArray allows to represent all of this Multi Modal data in a single object. 
+
+Let's first create an `BaseDoc` for each of the elements that compose the Youtube video.
+
+First for the thumbnail which is an image
+```python
+from docarray import BaseDoc
+from docarray.typing import ImageUrl, ImageBytes
+
+
+class ImageDoc(BaseDoc):
+    url: ImageUrl
+    bytes: ImageBytes = (
+        None  # bytes are not always loaded in memory, so we make it optional
+    )
+```
+
+Then for the video which is a video
+```python
+from docarray import BaseDoc
+from docarray.typing import VideoUrl, VideoBytes
+
+
+class ImageDoc(BaseDoc):
+    url: VideoUrl
+    bytes: VideoBytes = (
```

Comment:
same here

Reply:
No if you put `=None` it is optional by default